src/bindings/python/tests/test_frontend/test_frontend_onnx_editor.py

```diff
@@ -1643,7 +1643,7 @@ def test_add_the_same_name_to_tensor_twice():
 
     add_input = ov_model.input(1)
     add_input_tensor_names = add_input.get_names()
-    assert len(add_input_tensor_names) == 2
+    assert len(add_input_tensor_names) == 3
```

Comment:
Now the same string can be used several times as one of names of the same place. Is that purposely?

Reply:
No, but before we prohibited this behavior by using `unordered_set`. This PR changes `unordered_set` to `list` in order to support order of tensor names in the future.